pkg/proc/native/registers_windows_amd64.go

```diff
@@ -378,10 +378,7 @@ func registers(thread *Thread, floatingPoint bool) (proc.Registers, error) {
 	return regs, nil
 }
 
-type savedRegisters struct {
-}
-
-func (r *Regs) Save() proc.SavedRegisters {
+func (r *Regs) Copy() proc.Registers {
```

Comment:
We should just go ahead and implement this for Windows as well, get it all done in this pass.